packit_service/service/api/copr_builds.py

```diff
@@ -35,6 +35,13 @@
 ns = Namespace("copr-builds", description="COPR builds")
 
 
+def if_time_exists(datetime_object):
+    if datetime_object:
+        return datetime_object.strftime("%d/%m/%Y %H:%M:%S")
+    else:
+        return ""
```

Comment:
Would returning `None` here work?

`None` is a very good value to have, when you want to express a missing value. Much better then an empty string.

Reply:
Yes, will do.